### DISCUSSION TITLE
WinFSP issues

### DISCUSSION THREAD

**Discussion by KrystofHartel** [kubilus1/autoortho#33] [Q&A]
Hello i have problem with WinFSP does anyone know whats wrong?

![image](https://user-images.githubusercontent.com/55594490/211047865-6729d239-4b57-4138-b95b-88ec39ef7e3e.png)

**Comment by KrystofHartel** [comment]
![image](https://user-images.githubusercontent.com/55594490/211049994-b5cd9c33-b695-4bb8-b331-32fb0ade5ef6.png)

**Comment by ivotaHR** [comment]
> When using Windows I see an error using the run.bat file such as note: This error originates from a subprocess, and is likely not a problem with pip. error: legacy-install-failure

This is likely due to having a very new version of Python and a package dependency that does not have a pre-built 'wheel' binary. Further in that error you will see a link to visual studio build tools that can be installed. You could also try downgrading your version of Python. For instance try uninstalling Python 3.11 and install Python 3.10.

>  INSTALL  Python 3.10.    instead 3.11

**Reply by KrystofHartel** [reply]
thx

**Comment by kubilus1** [comment]
The error message shown is a python/pip message that tells you what you need to do if you wish to use the version of Python that you are on.

But I've been working with the package maintainer for WinFSP to include a pre-built package for Python 3.11 so this can be a bit smoother.  It should just work though, if you install the tools the error message suggests.

**Comment by mikecolive** [comment]
@kubilus1 After all set and installed, and when i press "run" within the app, i het an error saying "WinFSP module not found" in the cmd prompt.
I have installed WinFSP and python 3.11.
